conf.py

```diff
@@ -37,7 +37,7 @@
 
 # building the versions list
 version_start = 17		# THIS IS THE SUPPORTED VERSION NUMBER
```

Comment:
Well, I assume 17 is EOL now then?

Reply:
Thursday. So kinda yes

Reply:
Could you kinda approve the PR in the meantime to get the docs landing page updated for 20?